project_types/base/project.py

```diff
@@ -442,7 +442,13 @@ def update_project_on_firebase(self, project_ref: FbReference, fb_project: fireb
         )
 
     def _push_project_on_firebase(self):
-        if self.project.status_enum not in [Project.Status.READY_TO_PUBLISH, Project.Status.PUBLISHED]:
+        if self.project.status_enum not in [
+            Project.Status.READY_TO_PUBLISH,
+            Project.Status.PUBLISHED,
+            Project.Status.WITHDRAWN,
+            Project.Status.PAUSED,
+            Project.Status.FINISHED,
+        ]:
```

Comment:
As we have a lot of states now. We should instead add a whitelist instead of a blacklist.